pom.xml

```diff
@@ -525,10 +551,6 @@
       <version>1.21.3</version>
       <scope>test</scope>
       <exclusions>
-        <exclusion>
-          <groupId>org.apache.commons</groupId>
-          <artifactId>commons-compress</artifactId>
-        </exclusion>
```

Comment:
Need to remove this exclusion due to https://issues.jenkins.io/browse/JENKINS-73355